composables/masto/routes.ts

```diff
@@ -33,11 +33,11 @@ export function getReportRoute(id: string | number) {
   return `https://${currentUser.value?.server}/admin/reports/${encodeURIComponent(id)}`
 }
 
-export function getStatusRoute(status: mastodon.v1.Status) {
+export function getStatusRoute(status: mastodon.v1.Status, singleInstance?: boolean) {
   return useRouter().resolve({
     name: 'status',
     params: {
-      server: currentServer.value,
+      ...(!singleInstance && { server: currentServer.value }),
```

Comment:
If we are using singleInstance mode, we don't want the server included. The accountName on the status will be fully resolved 

- local user : `@somone/statusId`
- foreign user: `@somone@social.server/statusId`